### DISCUSSION TITLE
Can we modify it to work with gemini APIs

### DISCUSSION THREAD

**Discussion by whoabhaysharma** [antiwork/shortest#150] [Ideas]
Currently, this repo is designed to work with Claude APIs, which is great. However, I’d like to propose adding support for the Google Gemini API. Here's why:

Free Tier for Testing: Google Gemini offers a generous free usage tier, which could be super helpful for testing and experimentation without incurring additional costs.
Diversified Options: Supporting Gemini would provide more flexibility for developers to choose between APIs based on their use cases and resource availability.
Enhanced Capabilities: Gemini has advanced natural language understanding and generation features, which could potentially bring new capabilities to this project.
I think it’s worth exploring, especially for prototyping and evaluation purposes. What do you all think? Happy to contribute if we decide to move forward with this!

Looking forward to your thoughts.

**Comment by slavingia** [comment]
I don't think Gemini has a way to do "Computer Use" - but if they do (and I'm sure they, and OpenAI eventually will) then definitely!
